### PR TITLE
i did theStorage Run Command Skeleton

### LINKED ISSUE #13: Storage Run Command Skeleton
Loads the note into the scoring subsystem.

- [x]  Create a command named StorageRun.java. extend Command
- [x]  Do the import. import the storage subsystem
- [x]  in the constructor - add the storage subsystem as a requirement.
- [x]  add in the @ Override methods (execute, isFinished, end)
- [x]  execute should run the storage run methods
- [x]  leave isFinished blank. will Be updated with the addition of sensors
- [x]  end should run the storage stop methods.
- [x]  steal from pretty much any command.

### PR DESCRIPTION
**Pull request by Gaviniscool17**
### Pull Request

---------------------------------------------

#### Issue Number

Closes #13

#### Comments

I made the skeleton for the storage run command

### CODE REVIEW

**Review by Saum27 — APPROVED** [review]

**Review by matunkar — APPROVED** [review]

File: src/main/java/frc/robot/commands/StorageRunCommand.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Scoring;
```
Comment (reediculous456) [inline src/main/java/frc/robot/commands/StorageRunCommand.java:4]: Don't import scoring if it's not being used

File: src/main/java/frc/robot/commands/StorageRunCommand.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+public class StorageRunCommand extends Command {
+    private Storage storage = Storage.getInstance();
+
+    public StorageRunCommand() {
+        addRequirements(storage);
+    }
+
+    @Override
+    public void initialize() {
```
Comment (reediculous456) [inline src/main/java/frc/robot/commands/StorageRunCommand.java:15]: Remove this function since it's empty

**Review by reediculous456 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/Storage.java [hunk 3]
```diff
@@ -28,8 +28,8 @@ private Storage() {
         storageMotor.burnFlash();
     }
 
-    public void run() {
-        storageMotor.set(StorageConstants.motorSpeed);
```
Comment (reediculous456) [inline src/main/java/frc/robot/subsystems/Storage.java:32]: I'll leave this up to @matunkar as the controls lead, but either this needs put back and storage always runs at a constant speed OR the input speed needs to be clamped to a min and max value before being set

**Review by reediculous456 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/Storage.java (hunk 3 above)
Comment (reediculous456) [reply src/main/java/frc/robot/subsystems/Storage.java:32]: Thinking about how storage usually works, I believe this should be put back. Storage usually runs forward or backward at a constant speed in order to feed into acquisition consistently

**Review by reediculous456 — CHANGES_REQUESTED** [review]

**Comment by reediculous456** [comment]
My comments have not been addressed

**Pull request merged**